frontend/src/composables/useShootContext/index.js

```diff
@@ -124,10 +133,29 @@ export function createShootContextComposable (options = {}) {
     return normalizeManifest(object)
   })
 
+  function setExtension (shootManifest, type, extension) {
+    if (!Array.isArray(shootManifest?.spec?.extensions)) {
+      set(shootManifest, 'spec.extensions', [])
+    }
+    const extensions = shootManifest.spec.extensions
+    const index = findIndex(extensions, { type })
+
+    if (index === -1) {
+      extensions.push(extension)
+      return
+    }
+    if (extension) {
+      extensions[index] = extension
+    } else {
+      extensions.splice(index, 1)
+    }
```

Comment:
Somehow I overlooked the return statement. Either add a newline between the two if-cases or have it all in one if-else block
```go
    if (index === -1) {
      extensions.push(extension)
      return
    }

    if (extension) {
      extensions[index] = extension
    } else {
      extensions.splice(index, 1)
    }
```

```go
    if (index === -1) {
        extensions.push(extension);
    } else if (extension) {
        extensions[index] = extension;
    } else {
        extensions.splice(index, 1);
    }
```